src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -25,6 +25,19 @@ ascanalpha.examplefile.other=This is for information that doesnt fit in any of t
 ascanalpha.examplefile.soln=A general description of how to solve the problem
 ascanalpha.examplefile.refs=http://zaproxy.blogspot.co.uk/2014/04/hacking-zap-4-active-scan-rules.html
 
+ascanalpha.httponlysite.name = HTTP Only Site
+ascanalpha.httponlysite.desc = The site is only served under HTTP and not HTTPS.
+ascanalpha.httponlysite.soln = Configure your web or application server to use SSL (https).
+ascanalpha.httponlysite.refs = https://www.owasp.org/index.php/Transport_Layer_Protection_Cheat_Sheet\nhttps://www.owasp.org/index.php/SSL_Best_Practices\nhttps://letsencrypt.org/
+ascanalpha.httponlysite.otherinfo = {0}\nZAP attempted to connect via: {1}
+ascanalpha.httponlysite.otherinfo.connectionfail = Failed to connect.
+ascanalpha.httponlysite.otherinfo.differenthosts = Different Hosts.
+ascanalpha.httponlysite.otherinfo.redirectionlimit = Redirection limit reached.
+ascanalpha.httponlysite.otherinfo.noredirection = There was no automatic redirection.
+ascanalpha.httponlysite.otherinfo.nossl = Site has no SSL/TLS support.
+ascanalpha.httponlysite.otherinfo.redirecttohttp = Redirected to HTTP.
+ascanalpha.httponlysite.otherinfo.urinotencoded = URI Couldnt be encoded.
```

Comment:
Hopefully this is the last minor detail. There should be an apostrophe in couldn't, and it doesn't need leading caps.